bower.json

```diff
@@ -35,9 +35,9 @@
   ],
   "dependencies": {
     "firebase": "2.1.x",
-    "ember": "1.9.x",
     "ember-data": "1.0.0-beta.12"
   },
   "devDependencies": {
+    "ember": "1.9.x"
```

Comment:
Hmm, that sucks. We could be super explicit here and list `ember: 1.7.x || 1.8.x || 1.9.x` as a regular dependency. That seems like a better solution than putting `ember` in the dev dependencies. What do you think?